nf_core/pipeline-template/{{cookiecutter.name_noslash}}/.github/markdownlint.yml

```diff
@@ -3,3 +3,10 @@ default: true,
 line-length: false
 no-duplicate-header:
     siblings_only: true
+no-inline-html: 
+    allowed_elements:
+        - img
+        - p
+        - kbd
+        - details
+        - summary
```

Comment:
Will need to also add this to the main repo `.github/.markdownlint.yml` file so that the mdl linting on the tools repo passes.